terraform/workspace_variables/production.tfvars

```diff
@@ -18,3 +18,8 @@ statuscake_alerts = {
     node_locations = ["UKINT", "UK1", "MAN1", "MAN5", "DUB2"]
   }
 }
+
+key_vault_name           = "s121d01-shared-kv-01"
+key_vault_resource_group = "s121d01-shared-rg"
+key_app_secret_name      = "PUBLISH-APP-SECRETS-PRODUCTION"
+key_infra_secret_name    = "BAT-INFRA-SECRETS-PRODUCTION"
```

Comment:
key_vault_name and resource_group values are incorrect for the env, it should be `p01` 
same in other files as well

also can you rename `key_app_secret_name` and `key_infra_secret_name` to
`key_vault_app_secret_name ` and `key_vault_infra_secret_name ` to keep it consistent?

Reply:
ok... done.